diff.sh

```diff
@@ -86,7 +86,7 @@ if [[ $DIFF_OBJ = 1 ]]; then
     OBJDUMP="mips-linux-gnu-objdump -drz"
     $OBJDUMP $REFOBJFILE | grep "<$1>:" -A1000 > $BASEDUMP
     $OBJDUMP $OBJFILE | grep "<$1>:" -A1000 > $MYDUMP
-    DIFF_ARGS+=--diff-obj
+    DIFF_ARGS+=" -o --stop-jr-ra"
```

Comment:
I guess we should really have bash flags for this, or pass unused bash flags on to the diff script, but ... eh, I'm fine with being lazy.

Reply:
What do you mean bash flags? But i definitely agree with passing the unused bash flags.

Reply:
I having something like
```
-s)
  DIFF_ARGS+=" --stop-jr-ra"
  shift
```
in the switch at the top of the script. Passing on unused flags is slightly more complex because diff-settings.sh may also parse flags, so you need two switches, but on the whole it's not so bad.